website/docs/extensions/documentation-generation.md

```diff
@@ -1,6 +1,15 @@
 # Generating documentation
 
-`scarb doc` is a tool that generates documentation based on the code comments. Generation supports different output formats. The result is being placed inside the `/target/doc`.
+`scarb doc` is a tool that generates documentation based on the code comments. Generation supports different output formats. The result is being placed inside the `/target/doc` directory.
+
+### Generating workspace documentation
+You can run `scarb doc --workspace` to generate documentation for all packages in the workspace. 
+
+Use `--exclude` to omit the workspace packages in the documentation. Must be used with the `--workspace` flag. 
```

Comment:
Not sure if we want to advertise `--exclude` somewhere else in web documentation, wdyt?